CHANGELOG.asciidoc

```diff
@@ -35,6 +35,7 @@ https://github.com/elastic/apm-agent-go/compare/v1.8.0...master[View commits]
 - Add cloud metadata, configurable with ELASTIC_APM_CLOUD_PROVIDER {pull}823[#(823)]
 - Round ELASTIC_APM_SAMPLING_RATE with 4 digits precision {pull}828[#(828)]
 - module/apmhttp: implement io.ReaderFrom in wrapped http.ResponseWriter {pull}830[#(830)]
+- Fixed Transaction.Discard so that it sets TransactionData to nil {pull}836[#(836)]
```

Comment:
shouldn't this be under "bug fixes"?

Reply:
The Go agent changelog has not, to date, separated additions from fixes. There's a template comment above with a "bug fixes" section, but it's not used anywhere. Maybe it should be...

I'd like to defer that to when we release, and look at what the other agents are doing.